app/resonant-reference-app/models/Toolchain.js

```diff
@@ -134,7 +167,7 @@ let Toolchain = girder.models.ItemModel.extend({
     let options = {};
 
     // Figure out which options allow multiple fields
-    meta.mappings.forEach(mapping => {
+    meta.mappings.forEach((mapping) => {
```

Comment:
I think I went through and removed these parentheses at some point - do you think we should keep them or ditch them as a styling rule?